requirements.txt

```diff
@@ -1,7 +1,8 @@
-ase
-numpy
-requests
-flask
-flask-cors
-gunicorn
-pypif==1.1.6
+ase
+numpy
+requests
+flask
+flask-cors
+gunicorn
+pypif==1.1.6
+dftparse==0.1.2
```

Comment:
Is this already on pypi?

Reply:
Yes, [here](https://pypi.python.org/pypi/dftparse/0.1.2).  The tests would have failed if it wasn't.